components/artwork_metadata_stub/templates/contact.jade

```diff
@@ -1,10 +1,16 @@
-if artwork.is_in_auction
-  if artwork.is_biddable
-    .artwork-metadata-stub__contact.artwork-metadata-stub__line.artwork-metadata-stub__bid-now
+if artwork.sale && artwork.sale.is_auction
+  .artwork-metadata-stub__contact.artwork-metadata-stub__line.artwork-metadata-stub__bid-now
+    if artwork.sale.is_live_open
+      a(href= artwork.href) Enter Live Auction
+    else if artwork.sale.is_open
       a(href= artwork.href)
-        | Bid now
```

Comment:
I made a comment in the issue: https://github.com/artsy/force/issues/729#issuecomment-276114443 , but this might be an oversight? I think we should still be showing the 'Bid Now' link (in some cases) so as to link to the bid/artwork page.